src/renderer/atlas/AtlasEngine.cpp

```diff
@@ -502,6 +502,12 @@ try
         {
             for (const auto& ch : cluster.GetText())
             {
+                // Skip Unicode FSI (U+2068) and PDI (U+2069).
+                // TODO: #538 - Remove this once we have proper support for bidirectional text.
+                if (ch == 0x2068u || ch == 0x2069u)

```

Comment:
Shouldn't we be doing this for the whole range of isolate characters: U+2066 through U+2069? They're all affected by this issue, and the test wouldn't be any more complicated.